prow/cmd/peribolos/main.go

```diff
@@ -424,10 +424,62 @@ func configureTeams(client teamClient, orgName string, orgConfig org.Config) (ma
 	return matches, nil
 }
 
+// updateString will return true and set have to want iff they are set and different.
+func updateString(have, want *string) bool {
+	if want == nil || have == nil || *have == *want {
```

Comment:
in what case would `have == nil`? If Github doesn't return the field at all?

Reply:
Done

Reply:
So as these methods are presently used, never. But logically it is possible to call updateString with nil have, and I want to handle this situation rather than blowing up with a nil pointer exception